docs/operation/upgrade.rst

```diff
@@ -0,0 +1,100 @@
+Upgrade Guide
+=============
+This document describes a complete and reliable upgrade path for **MetalK8s**
+and all the components that make up the stack.
+
+.. important::
+
+    - Upgrading MetalK8s is a manual process and as a result,
+      the following outlined steps must be met to the later in other to achieve
+      a successful upgrade.
+    - The output of the below-provided commands solely depends on the particular
+      MetalK8s setup you have in place.
+    - **<new_version>** and **<current_version>** are suffixes which must be
+      replaced with appropriate version number.
+
+Supported Versions
+******************
+.. note::
+
+    MetalK8 supports upgrades and downgrades **strictly** from one supported
+    minor version to another. For example;
+
+    - Upgrade from 2.0.0 to 2.0.1
+    - Upgrade from 2.0.0 to 2.1.0
+
+    Please refer to the release notes for more information.
+
+Upgrade Steps
+*************
+This section describes the logical steps for Upgrading MetalK8s.
+
+#. Connect to the :term:`Bootstrap node`.
+
+#. Build and copy the new MetalK8 `.ISO` file to the **Bootstrap node**.
+
+#. Edit the :file:`bootstrap.yaml` file located in
+   :file:`/etc/metalk8s/bootstrap.yaml` then add the new MetalK8s ISO location.
+
+   .. code::
+
+      products:
+        metalk8s:
+        - /srv/scality/metalk8s-2.0.0-dev
+        - <path_to_the_new_iso>              <=== New line to add
+
+
+#. Mount the new MetalK8s ISO.
+
+    ::
+
+        salt-call state.sls metalk8s.products.mounted saltenv=metalk8s-<current_version>
+
+            Summary for local
+            ------------
+            Succeeded: 5 (changed=1)
+            Failed:    0
+            ------------
+            Total states run:     5
+
+#. Make the new version available to the entire cluster.
+
+    ::
+
+        salt-call --local state.sls metalk8s.products saltenv=metalk8s-<new_version> pillar="{'metalk8s': {'endpoints': $(salt-call --out txt pillar.get metalk8s:endpoints | cut -c 8-)}}"
+
+#. From the :term:`Bootstrap node`, connect to the :term:`Salt Master`.
+
+    ::
+
+        crictl ps -a   #List all containers
+
+    ::
+
+        crictl exec -it <salt_master_container_id> sh  #Connect to salt-master
```

Comment:
I think you should use the one line command I gave you so the user can just "copy/paste" it
```
crictl exec -it $(crictl ps -q --label io.kubernetes.container.name=salt-master) bash
```

Reply:
ok.. merci